hubclient_lib/conf.py

```diff
@@ -33,9 +34,8 @@ def __init__(self):
     def __getattr__(self, key):
         try:
             return self[key]
-        except KeyError, e:
+        except KeyError as e:
             raise AttributeError(e)
```

Comment:
This should probably be

```python
try:
    return self[key]
except KeyError as e:
    raise AttributeError(
        f'{self.__class__.__name__!r} object has no attribute {key!r}',
        name=key, obj=self) from e
```

Because:

`AttributeError.args[0]` should be a string description, not an exception
`AttributeError.name` should be the name of the invalid attribute, but is `None` when not set
`AttributeError.obj` should be the object that failed validation, but is `None` when not set

Reply:
Thanks again! :smile: